Desktop_Interface/isobuffer.cpp

```diff
@@ -109,15 +124,13 @@ void isoBuffer::writeBuffer(T* data, int len, int TOP, Function transform)
 	{
 		bool isUsingAC = m_channel == 1
 		                 ? m_virtualParent->AC_CH1
-						 : m_virtualParent->AC_CH2;
+		                 : m_virtualParent->AC_CH2;
 
-		for (int i = 0; i < len; i++)
+		for (int i = 0; i < len && m_fileIOEnabled; i++)
```

Comment:
Hadn't considered the case where the user aborts the file mid-write.  Nice.

Reply:
It also stops the loop once the file size limit is reached.